### ISSUE TITLE
Implement ToolCommandlet for Python

### ISSUE THREAD

**Issue by hohwille** [devonfw/IDEasy#38]
Create `ToolCommandlet` for Python
Old commandlet:
https://github.com/devonfw/ide/blob/master/scripts/src/main/resources/scripts/command/python

ATTENTION: This story is blocked by #352 that needs to be implemented first. UPDATE: #352 is done so unblocked.

See also #352 that needs to be considered when implementing this story.

See also #353 (pip)

**Comment by julia-cap** [comment]
This support for Python on Linux will be handled in scope of another issue. This issue includes support for Windows and Mac only.
